_posts/2021-08-19-Showcase.md

```diff
@@ -11,11 +11,11 @@ By Josephine Watkins
 
 Over 40 researchers and campus research computing staff were selected to attend this year’s OSG Virtual School, all united by a shared desire to learn how high throughput computing can advance their work. During the first two weeks of August, school participants were busy attending lectures, watching demonstrations, and completing hands-on exercises; but on Wednesday, August 11, participants had the chance to hear from researchers who have successfully used high throughput computing (HTC) to transform their work. Year after year, this event –– the HTC Showcase –– is one highlight of the experience for many User School participants. This year, three different researchers in the fields of structural biology, psychology, and particle physics shared how HTC impacted their work. Read the articles below to learn about their stories.
 
-**[Scaling virtual screening to ultra-large virtual chemical libraries]()** – Spencer Erickson, Carbone Cancer Center, University of Wisconsin-Madison
+**[Scaling virtual screening to ultra-large virtual chemical libraries](https://opensciencegrid.org/news/2021/08/19/Spencer-Showcase.html)** – Spencer Erickson, Carbone Cancer Center, University of Wisconsin-Madison
```

Comment:
@josiewatkins: It’s “Ericksen” — there’s no “o”. Please check all occurrences.

Reply:
Will do, thanks

Reply:
Corrected in all occurrences! If you could go ahead and merge that would be great @osg-cat